var/spack/repos/builtin/packages/votca/package.py

```diff
@@ -39,9 +39,10 @@ class Votca(CMakePackage):
     depends_on("expat")
     depends_on("fftw-api@3")
     depends_on("eigen@3.3:")
-    depends_on("boost@1.71:+filesystem+system+regex+timer")
-    depends_on("boost@1.71:1.84+filesystem+system+regex+timer", when="@2024")
-    depends_on("boost@1.71:1.82+filesystem+system+regex+timer", when="@:2023")
+    depends_on("boost+filesystem+system+regex+timer")
+    depends_on("boost@1.71:")
+    depends_on("boost@1.71:1.84", when="@2024")
```

Comment:
I am always confused, does `@2024` means just `v2024` or `v2024.*`? We are about to make the `v2024.1`, which will have the fix to build with boost-1.85.